region/norfair/east/Upper Norfair Farming Room.json

```diff
@@ -318,6 +331,28 @@
         {"heatFrames": 250}
       ]
     },
+    {
+      "link": [5, 5],
+      "name": "Wave Beam the Gate",
+      "notable": false,
+      "requires": [
+        "h_canNavigateHeatRooms",
+        {"heatFrames": 120},
+        "Wave"
+      ],
+      "clearsObstacles": ["A"]
+    },
+    {
```

Comment:
I'm a little surprised these two have the same heat frames as well, since you can shoot wave through the top easily, whereas gate glitching you need to get into position next to the gate. Maybe it's right because of the heat frames in `h_canHeatedBlueGateGlitch`

Reply:
These counts must be left over from when the strat continued to the door.  The number should be very small with wave now.